README.md

```diff
@@ -1 +1,42 @@
 # Advanced-Building-Blocks---Enumerables
+
+In this project, I've built my own version of the built-in Ruby’s enumerable methods:
+
+- Each;
+- Each with index;
+- Select;
+- All;
+- Any;
+- None;
+- Count;
+- Map;
+- Inject.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+- open CMD
+- git clone https://github.com/AbdelrhmanAmin/Advanced-Building-Blocks---Enumerables.git
+- **Make sure Ruby is installed on your computer**
+- in terminal type: ruby index.rb
```

Comment:
- [ ] **OPTIONAL** If you would like to format your terminal instructions as code, you can use backquotes
```
`like this`
```